packages/graphql/mixin.core.js

```diff
@@ -41,11 +42,15 @@ class GraphQLMixin extends Mixin {
               headers: argv.header,
             })
               .then(() => {
-                console.log('Fetched and saved GraphQL fragments');
+                if (typeof this.getLogger === 'function') {
+                  this.getLogger.info('Fetched and saved GraphQL fragments');
+                }
               })
               .catch(err => {
-                console.error('Could not fetch GraphQL fragments:');
-                console.trace(err);
+                if (typeof this.getLogger === 'function') {
+                  this.getLogger.error('Could not fetch GraphQL fragments:');
+                  console.trace(err);
```

Comment:
I would prefer us to come up with a different solution here - maybe just call  `this.getLogger().error()` twice? It does have `Error` object handling built in, after all.